Kingfisher/UIImageView+Kingfisher.swift

```diff
@@ -104,12 +106,30 @@ public extension UIImageView {
     
     :returns: A task represents the retriving process.
     */
+    
     public func kf_setImageWithURL(URL: NSURL,
                       placeholderImage: UIImage?,
                            optionsInfo: KingfisherOptionsInfo?,
                          progressBlock: DownloadProgressBlock?,
                      completionHandler: CompletionHandler?) -> RetrieveImageTask
     {
+        
+        var indicator = UIActivityIndicatorView(activityIndicatorStyle:.Gray)
```

Comment:
Indicator should not be created twice. I believe you only won't to create it inside the if below